### PR TITLE
User gallery layout

### PR DESCRIPTION
**Pull request by Amstutz**
Promised PR to match the implementation closer to the mockups provided by Yvonne Seiler for Request [http://www.ilias.de/docu/goto_docu_wiki_wpage_2003_1357.html](http://www.ilias.de/docu/goto_docu_wiki_wpage_2003_1357.html). 

- Flex is used to render all the cards in the same height. 
- The background color was changed to set some contrast to the background of the parent.
- The Username and the responsibility was moved into a descriptive list.

### CODE REVIEW

File: Services/User/Gallery/classes/class.ilUsersGalleryGUI.php
Comment (mjansenDatabay) [inline Services/User/Gallery/classes/class.ilUsersGalleryGUI.php]: I like the idea of using a listing here!

File: Services/User/Gallery/classes/class.ilUsersGalleryGUI.php
Comment (mjansenDatabay) [inline Services/User/Gallery/classes/class.ilUsersGalleryGUI.php]: I would prefer a single space between the array key, the assignment operator, and the value.

File: src/UI/templates/default/Card/card.less
Comment (mjansenDatabay) [inline src/UI/templates/default/Card/card.less]: I like the flexible box layout approach. IE 11 seems to only partly support it (see: https://caniuse.com/#search=flex), but we should force users to use a _state of the art_ browser ;-).

**Review by mjansenDatabay — APPROVED** [review]

File: src/UI/templates/default/Card/card.less
Comment (Amstutz) [reply src/UI/templates/default/Card/card.less]: Did we make a decision yet concerning support of 5.3 for IE 11?

**Comment by klees** [comment]
Sry, @smeyer-ilias is the maintainer of the User-service, not @alex40724.

File: src/UI/templates/default/Deck/deck.less
Comment (alex40724) [inline src/UI/templates/default/Deck/deck.less:23]: a row-eq-height class name sounds like a general approach to make the bootstrap col divs in a bootstrap row of equal height (which would be good to have, if possible). But this would move the responsibility to define the class in the general delos.less (or some other more central place). If deck.less defines this class, the name should be il-deck-row-eq-height or something imo.

**Comment by mjansenDatabay** [comment]
@Amstutz @alex40724 @smeyer-ilias 

What is the status of this PR? In my opinion this should be merged because it improves the UI of the following feature a lot: https://www.ilias.de/docu/goto_docu_wiki_wpage_4328_1357.html

In my opinion we don't have to wait for Stefan's approval, because the complete member gallery code was revised with 5.1.x (and now further enhanced with 5.3.x) by @gvollbach and @mjansenDatabay .

**Comment by alex40724** [comment]
Yes, if you two and @Amstutz agree this can be merged imo.

**Comment by mjansenDatabay** [comment]
@Amstutz Any objections?

**Pull request merged**
